inst/extdata/PA2022CH_en_exec_summary/template.Rmd

```diff
@@ -250,26 +255,21 @@ plot_alignment_table(data_alignment_table)
 
 \newpage
 
-```{r conditional_print, child='survey-chapter.Rmd', eval = (dir.exists(survey_dir) & length(list.files(survey_dir)) > 0)}
+```{r survey_chapter, child='survey-chapter.Rmd', eval = (dir.exists(survey_dir) & length(list.files(survey_dir)) > 0)}
 ```
 
 ## Real Estate & Mortgage Results
 
-The Real Estate and Mortgage section allows financial institutions to measure the alignment of their Swiss real estate and mortgage portfolio with the climate target of the Swiss building sector to reach net-zero emissions in 2050. Therefore, the PACTA test assesses the CO2 emissions of your real estate and mortgage investments.<br/>
-Emissions from the Swiss building stock currently account for just over a quarter of Switzerland's total CO2 emissions. New buildings are largely equipped with renewable heating systems today. By 2020, CO2 emissions from buildings as a whole should have fallen by at least 40% below 1990 levels - this target has been missed. In order to achieve the net zero target declared by the Federal Council in 2050, Switzerland's building stock must reduce its CO2 emissions more rapidly in the future, in particular by renovating existing buildings. ([Link](https://www.bafu.admin.ch/bafu/en/home/topics/climate/info-specialists/emission-reduction/reduction-targets/2050-target/climate-strategy-2050.html))
-
-
 ### Current status
```

Comment:
these four lines are astray and should be in line 27f of the real estate chapter

Reply:
Already in there.